Lanthanum-web/Controllers/HomeController.cs

```diff
@@ -19,12 +19,15 @@ public class HomeController : Controller
         public HomeController(ILogger<HomeController> logger)
         {
             _logger = logger;
+            Categories = new List<string>
```

Comment:
Maybe enum will be better decision.

Reply:
But then there are problems with adding/removing categories by admin

Reply:
Ok, I agree with you. You have my approve.